controllers/machine_controller.go

```diff
@@ -83,6 +89,47 @@ func (r *MachineReconciler) SetupWithManager(mgr ctrl.Manager, options controlle
 		return errors.Wrap(err, "failed setting up with a controller manager")
 	}
 
+	mapFn := handler.ToRequestsFunc(
+		func(a handler.MapObject) []reconcile.Request {
+			return []reconcile.Request{
+				{NamespacedName: types.NamespacedName{
+					Name:      a.Meta.GetName(),
```

Comment:
This is the cluster's name, which isn't what we want to enqueue. I think you'll need to list all the machines for the cluster and return their names.

Reply:
getting all machine in the cluster